src/modules/tests/test-status/test-status.reducer.ts

```diff
@@ -37,7 +37,9 @@ export function testStatusReducer(state = initialState, action: testStatusAction
         [action.slotId]: TestStatus.Submitted,
       };
     default:
-      return state;
+      return {
+        ...state,
+      };
```

Comment:
We don't want this change.

DeepClone before sending the test data back and remove all null attributes from that new object and send that object. Basically leave the Store's state untouched from outside of reducers.